requirements/requirements.txt

```diff
@@ -4,14 +4,14 @@ watchdog>=2.1, <3.0
 combo-lock>=0.2.2, <0.3
 
 padacioso>=0.2.2,<1.0.0
-ovos-adapt-parser>=0.1.1, <1.0.0
-ovos_ocp_pipeline_plugin>=0.1.0, <1.0.0
-ovos-common-query-pipeline-plugin>=0.1.0, <1.0.0
+ovos-adapt-parser>=0.1.2, <1.0.0
+ovos_ocp_pipeline_plugin>=0.1.2, <1.0.0
+ovos-common-query-pipeline-plugin>=0.1.2, <1.0.0
 
 ovos-utils>=0.1.0,<1.0.0
 ovos_bus_client>=0.1.0,<1.0.0
 ovos-plugin-manager>=0.0.26,<1.0.0
 ovos-config>=0.0.13,<1.0.0
 ovos-lingua-franca>=0.4.7,<1.0.0
 ovos-backend-client>=0.1.0,<2.0.0
-ovos-workshop>=0.0.16,<1.0.0
+ovos-workshop>=1.0.0,<2.0.0
```

Comment:
TODO: minimum workshop version will be reverted to 0.0.16  just making sure everything also works with 1.0.0